TROUBLESHOOTING.md

```diff
@@ -1,6 +1,6 @@
 # Troubleshooting
 
-## Logging
+## Advanced Logging Setup
 
 Our libraries use the Java logging API via `java.util.logging` package.
```

Comment:
I think a lot of the content needs to be updated along with the title. For example, `Our libraries use the Java logging API` is not valid anymore, we should say something like "our core dependencies and some handwritten libraries use the Java logging API".

Reply:
Updated the contents, please take a look.